docs/source/markdown/podman-run.1.md

```diff
@@ -1848,6 +1848,25 @@ $ podman run --name container1 --personaity=LINUX32 fedora bash
 $ podman run --name container1 --rootfs /path/to/rootfs:O bash
 ```
 
+### Handling Timezones in java applications in a container.
+
+In order to use a timezone other than UTC when running a
+Java application within a container, the `TZ` environment variable must be
+set within the container. Java applications will ignore the value set with the
+`--tz` option.  This will be corrected in a later relase.
```

Comment:
Take out "This will be corrected in a later relase." (Spelling anyways.)  Don't make promises in man pages.

Reply:
@rhatdan Tom is out for the rest of the week. Should we get this in and I can open up a follow up PR to remove the "This will be corrected in a later release" line.

Reply:
Yes.